src/components/FormSectionHeader/FormSectionHeader.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Grid, makeStyles } from '@material-ui/core';
+
+const borderColor = '#404040';
+const useStyles = makeStyles({
+  titleBox: {
+    fontFamily: 'Avenir Next',
+    backgroundColor: borderColor,
+    color: 'white',
+    border: `3px solid ${borderColor}`,
+    letterSpacing: 1,
+    display: 'inline-block'
+  },
+  line: {
+    display: 'inline-block',
+    border: `1px solid ${borderColor}`,
+    width: '100%'
+  }
+});
+
+function FormSectionHeader({ title }) {
+  const classes = useStyles();
+
+  return (
+    <Grid container direction="row">
+      <Grid item>
+        <div className={classes.titleBox}>{title}</div>
```

Comment:
I think this will have to be a header (e.g. `<h1 />`) for tocbot to pick it up

Reply:
Not necessarily. Tocbot can be configured to use ids or class names.